CD-faqs (#7496)
harness/developer-hub: docs/continuous-delivery/deploy-srv-diff-platforms/azure/azure-faqs.md

### How can one publish a Maven artifact to the artifact feed instead of another repository in Harness?

One can modify the `pom.xml` file to configure the mvn publish command to push to the desired artifact feed. This might involve changing the repository settings in the `pom.xml` to point to the correct artifact feed.

> ### Is user can do the AZURE_VMSS deployment through Harness?

Reviewer: ```suggestion
### Can I do an AZURE_VMSS deployment through Harness?
```
